src/sentry/api/event_search.py

```diff
@@ -794,6 +794,15 @@ def convert_search_filter_to_snuba_query(search_filter, key=None):
                 1,
             ]
         return [user_display_expr, search_filter.operator, value]
+    elif name == "error.handled":
+        # null values and 1 are the same, and both indicate a handled error.
+        if value in ("1", 1):
+            return [["isHandled", []], "=", 1]
```

Comment:
What happens if I do `!error.handled:1`?

Reply:
You would get an error as you can't ! numeric fields. The tests pointed out that I forgot about has as well and I'll get that fixed.